start.sh

```diff
@@ -90,6 +90,12 @@ if [[ -z "$EMQ_LISTENER__WS__EXTERNAL__MAX_CLIENTS" ]]; then
     export EMQ_LISTENER__WS__EXTERNAL__MAX_CLIENTS=250000
 fi
 
+# Fix issue #42 - export env EMQ_DASHBOARD__DEFAULT_USER__PASSWORD to configure
+# 'dashboard.default_user.password' in etc/plugins/emq_dashboard.conf
+if [[ ! -z "$EMQ_ADMIN_PASSWORD" ]]; then
```

Comment:
This doesn't solve the problem because in some cases the plugins can take more than five seconds to load. Can I suggest force the update:

```sh
if [[ ! -z "$EMQ_ADMIN_PASSWORD" ]]; then
  PASSWORD_CHANGE_TRY=0
  while [[ $PASSWORD_CHANGE_TRY -lt 5 ]]
  do
    PASSWORD_CHANGED=$(/opt/emqttd/bin/emqttd_ctl admins passwd admin $EMQ_ADMIN_PASSWORD | grep -w 'ok')
    if [[ ! -z "$PASSWORD_CHANGED" ]]; then
      echo "['$(date -u +"%Y-%m-%dT%H:%M:%SZ")']:admin password changed to $EMQ_ADMIN_PASSWORD"
      break
    else
      echo "['$(date -u +"%Y-%m-%dT%H:%M:%SZ")']:admin password not changed: trying again"
      PASSWORD_CHANGE_TRY=$((PASSWORD_CHANGE_TRY+1))
      sleep 5
    fi
  done

  if [[ -z "$PASSWORD_CHANGED" ]]; then
      echo "['$(date -u +"%Y-%m-%dT%H:%M:%SZ")']:admin password change failed"
      exit 1
  fi
fi
```